links de plataformas de ícones grátis

mggcode opened the pull request:
links de plataformas de ícones grátis

mggcode committed:
Cursos Harvard

mggcode committed:
changing course place

mggcode committed:
fundação estudarcc50

mggcode committed:
captain icons

mggcode committed:
putting in alphabetical order

mggcode committed:
round icon and captain icon

mggcode committed:
linearicons

mggcode committed:
pixeden link

mggcode committed:
fontello

mggcode committed:
the noun project link

mggcode committed:
freebie supply

iuricode commented:
Olá Marta!

Obrigado pela contribuição no projeto Recursos Gratuitos. Só gostaria de solicitar uma coisa para aceitar seu Pull Request, consegueria adicionar essa alterações nos outros README? Isso é, de Inglês, Espanhol e Mandarim.

Até, Iuri Silva.

mggcode committed:
novos links icones

mggcode committed:
arrumando o link de harvard

mggcode committed:
links de harvard e fundação estudar

mggcode committed:
link fundação estudar e harvard

mggcode committed:
link harvard e estudar fora

mggcode committed:
links

links Round Icon, Pixeden, The noum project, Linearicon, Free Icons, Fontello, Captain Icons

mggcode committed:
links

links Captain Icons, Fontello, Free icons, Linearicons, Pixeden, Round Icon, The noun Project

mggcode commented:
adicionado Iuri, não entendo mandarim, mas fui pela que estava escrito nos outros! ; )

mggcode committed:
link IFRS

mggcode committed:
link IFRGS

mggcode committed:
link Instituto Federal do Rio Grande do Sul

mggcode committed:
link do Instituto Federal do Rio Grande do Sul

mggcode commented:
link do Instituto Federal do RS
